storybook/native/main.js

```diff
@@ -1,34 +1,113 @@
-import React from 'react';
-import { AppRegistry } from 'react-native';
+import React, { useCallback } from 'react';
+import { AppRegistry, StatusBar } from 'react-native';
 import { getStorybookUI, configure, addDecorator } from '@storybook/react-native';
 import styled, { ThemeProvider } from 'styled-components';
-
-import theme from '../../src/tokens/theme';
+import { darkTheme, lightTheme } from '../../src/tokens/theme';
 import './rn-addons';
+import AsyncStorage from '@react-native-community/async-storage';
+import storybookTheme from './storybookTheme';
+
+const theme = {
+  light: lightTheme,
+  dark: darkTheme,
+};
 
 // import stories
 configure(() => {
   require('../../src/atoms/Button/Button.stories');
 }, module);
 
+// add decorators
 const SpaceAround = styled.View`
   margin: 20px;
 `;
 
 addDecorator((Story) => (
-  <ThemeProvider theme={theme}>
-    <SpaceAround>
-      <Story />
-    </SpaceAround>
-  </ThemeProvider>
+  <SpaceAround>
+    <Story />
+  </SpaceAround>
 ));
 
-// Refer to https://github.com/storybookjs/storybook/tree/master/app/react-native#start-command-parameters
-// To find allowed options for getStorybookUI
-const StorybookUIRoot = getStorybookUI({});
+const SafeAreaWrapper = styled.SafeAreaView`
+  flex: 1;
+  background-color: ${(props) =>
+    props.isDarkTheme ? theme.dark.colors.background[400] : theme.light.colors.background[400]};
+`;
+
+const ThemeSwitch = styled.Switch`
+  margin-left: 10px;
+  margin-right: 10px;
+`;
+
+const SwitchContainer = styled.View`
+  width: ${'100%'};
```

Comment:
you don't need `${}` since it's already inside template literals

Reply:
Done